cmd/vcfFilter/vcfFilter.go

```diff
@@ -282,6 +298,7 @@ func main() {
 	var infoExp *string = flag.String("info", "", "Identical to the 'format' tag, but tests the info field. The values of type 'Flag' in the info field"+
 		"can be tested by including just the flag ID in the expression. E.g. To select all records with the flag 'GG' you would use the expression \"GG\".")
 	var includeMissingInfo *bool = flag.Bool("includeMissingInfo", false, "When querying the records using the \"-info\" tag, include records where the queried tags are not present.")
+	var subSet *float64 = flag.Float64("subSet", 1, "Proportion of variants to retain in output. Value must be between 0 and 1.")
```

Comment:
Would it be helpful to add to the message here, saying that in order to get a small set of variants from a large vcf file, the recommended action is to first subset with vcfFilter, then feed the result into sampleVcf?